components/ReactBKKLogo/RevealEllipse.js

```diff
@@ -44,6 +45,15 @@ class RevealEllipse extends React.Component {
       this.timer = Date.now()
       this.raf = window.requestAnimationFrame(this.revealStep)
     }, this.props.delay)
+    document.getElementsByTagName('BODY')[0].addEventListener('click', function (e) {
```

Comment:
Sorry, this is not the right place to put this code. This is just an ellipse inside the React Bangkok logo. If we remove the logo from the page, or replace it with some other component, smooth scrolling will not work.

Please check out the comments in #81 for further suggestions!